src/RepoM.Api/Bootstrapper.cs

```diff
@@ -1,38 +1,30 @@
 namespace RepoM.Api;
 
-using Microsoft.Extensions.Logging;
-using RepoM.Api.Common;
-using RepoM.Api.IO;
-using RepoM.Api.Plugins;
-using SimpleInjector;
+using System;
 using System.Collections.Generic;
 using System.IO.Abstractions;
 using System.Linq;
 using System.Reflection;
 using System.Threading.Tasks;
-using System;
+using Microsoft.Extensions.Logging;
+using RepoM.Api.Common;
+using RepoM.Api.IO;
+using RepoM.Api.Plugins;
 using RepoM.Api.Plugins.SimpleInjector;
 using RepoM.Core.Plugin;
 using RepoM.Core.Plugin.Common;
 using RepoM.Core.Plugin.RepositoryOrdering.Configuration;
+using SimpleInjector;
 
-public class CoreBootstrapper
+public class CoreBootstrapper(IPluginFinder pluginFinder, IFileSystem fileSystem, IAppDataPathProvider appDataProvider, ILoggerFactory loggerFactory)
```

Comment:
Please revert this convention / code style. It is only making the PR review difficult. I don't like this style. I'm aware of the inconsistency of the code styling but I would like to address this in a different PR and do it right over the whole project.